components/brave_vpn/features.h

```diff
@@ -12,6 +12,7 @@ namespace brave_vpn {
 namespace features {
 
 BASE_DECLARE_FEATURE(kBraveVPN);
+BASE_DECLARE_FEATURE(kBraveVPNDnsProtection);
```

Comment:
is_win buildflag

Reply:
added in https://github.com/brave/brave-core/pull/15989/commits/08d26ca84c8bb4831ece6860cc0d35eacaa6328f